index.js

```diff
@@ -63,8 +63,6 @@ function getTarget (abi, runtime) {
 }
 
 var supportedTargets = [
-  {runtime: 'node', target: '0.10.48', abi: '11', lts: false},
-  {runtime: 'node', target: '0.12.17', abi: '14', lts: false},
```

Comment:
Hmm, if we add these to deprecatedTargets as well, then wouldn't we have to deprecate all other 0.10 and 0.12 versions as well? It's essentially the range 0.10.x and 0.12.x we want to deprecate, right?